### PR TITLE
doc bash 2-5, for pack, unpack, pack_img and unpack_img

### PR DESCRIPTION
**Pull request by jiayue666**
Doc Bash for 2-5. i.e. mx.recordio.pack, mx.recordio.unpack, mx.recordio.pack_img, and mx.recordio.unpack_img. 
@mli @zackchase @madjam @nswamy

### CODE REVIEW

File: python/mxnet/recordio.py [hunk 1]
```diff
@@ -282,9 +282,23 @@ def pack(header, s):
...
+        Raw image string to be packed.
+
+    Returns
+    -------
+    s : str
+        The packed string.
+
+    Examples
+    --------
+    >>> label = 4
```
Comment (piiswrong) [inline python/mxnet/recordio.py]: label can also be a 1d number array.

**Comment by jiayue666** [comment]
Hi Eric @piiswrong , I update the code based on your suggestion. Please review when you are free

**Comment by piiswrong** [comment]
please fix lint

**Comment by Roshrini** [comment]
Hi @jiayue666, Thanks for the contribution. Can you please fix the lint error so that this PR can get merged?

**Comment by jiayue666** [comment]
@Roshrini Sure. I will update this ASAP. Sorry for late fixing, since I got too many staff this week.

**Comment by jiayue666** [comment]
@Roshrini @piiswrong fixed lint

File: python/mxnet/recordio.py [hunk 2]
```diff
@@ -340,14 +362,35 @@ def unpack_img(s, iscolor=-1):
...
+    >>> header, img = mx.recordio.unpack_img(item)
+    >>> header
+    HEADER(flag=0, label=14.0, id=20129312, id2=0)
+    >>> img
+    array([[[ 23,  27,  45],
+            [ 28,  32,  50],
+            ...,
+            [ 36,  40,  59],
+            [ 35,  39,  58]],
+
```
Comment (piiswrong) [inline python/mxnet/recordio.py]: I remember someone said blank lines won't render well. Have you verified that?

**Pull request merged**